python/tests/test_lowlevel.py

```diff
@@ -445,6 +457,30 @@ def test_mean_descendants(self):
             A = ts.mean_descendants([focal[2:], focal[:2]])
             self.assertEqual(A.shape, (ts.get_num_nodes(), 2))
 
+    def test_metadata_schemas(self):
+        tables = _tskit.TableCollection(1.0)
+        # Set the schema
+        for table_name in self.metadata_tables:
+            getattr(
```

Comment:
I'd find this more readable as 
```python
table = getattr(tables, f"{table_name}s")
table.metadata_schema = f"{table_name} test metadata schema"
```
Had to read it back a couple of times to figure out what it was doing.

Reply:
Yeah, line breaks don't help. Fixed in https://github.com/tskit-dev/tskit/pull/542/commits/fc229a4e3687baa3ba025123add768a2597efacb